### DISCUSSION TITLE
Cycle drawing

### DISCUSSION THREAD

**Discussion by JulianRodriguez10** [CoolProp/CoolProp#2366] [General]
Hola, no entiendo bien como puedo realizar el dibujo de ciclos si parto con un grupo de estados de la forma cycle_states = StateContainer(). ¿xiste alguna manera?

**Comment by mBelisarius** [comment]
The code below is an example to plot the process using a `StateContainer`. Change the `cycle_states` and the fluid in the `PropertyPlot` corresponding to your problem.

You can search for each parameter of `PropertyPlot` in the [CoolProp documentation](http://www.coolprop.org/apidoc/CoolProp.Plots.Plots.html).

```
cycle_states = StateContainer()
plot = PropertyPlot('Water', 'TS', unit_system='kSI')

plot.calc_isolines(CP.iQ, num=20)
plot.calc_isolines(CP.iP, num=20)

plot.draw_process(cycle_states, line_opts={'color': 'blue', 'lw': 1.5})

plot.title(r'Temperature-Entropy (T-s) Diagram')
plot.xlabel(r'$s$ [kJ/kg·K]')
plot.ylabel(r'$T$ [K]')

plot.show()
plot.savefig('diagram.svg')
```

**Comment by JulianRodriguez10** [comment]
Thank you very much for the help, I appreciate it very much.
